qa/L0_tf_gpu_io/test.sh

```diff
@@ -40,7 +40,7 @@ fi
 
 export CUDA_VISIBLE_DEVICES=0
 
-CLIENT=../clients/perf_client
+TF_TEST=tf_gpu_io_test.py
 BACKENDS=${BACKENDS:="graphdef savedmodel"}
 TENSOR_SIZE=16384
```

Comment:
This should either be removed or read in as an env var in the python test

Reply:
Good catch. Removed.